package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "dp186_lms",
+    "version": "1.0.0",
+    "description": "Learning management system",
+    "main": "app.js",
```

Comment:
this should be changed to the initial file, transpiled from typescript

Reply:
done